src/main/java/net/kaczmarzyk/spring/data/jpa/web/StandaloneProcessingContext.java

```diff
@@ -0,0 +1,100 @@
+/**
+ * Copyright 2014-2022 the original author or authors.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ * http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+package net.kaczmarzyk.spring.data.jpa.web;
+
+import net.kaczmarzyk.spring.data.jpa.utils.QueryContext;
+
+import java.lang.annotation.Annotation;
+import java.util.Map;
+
+import static java.util.Objects.nonNull;
+
+/**
```

Comment:
add comment about purpose of this class (when is it used = outside web layer). Also link to the specification builder class